src/sordid-dragon.jquery.coffee

```diff
@@ -61,20 +61,32 @@ do ($=jQuery) ->
     # create a ghost under the cursor for us. That doesn't happen with touch
     # events, so we create one ourselves.
     $ghost = null
+    $ghostContainer = $("<div />")
+      .addClass("ghost-container")
+      .css
+        display: "none"
+        position: "fixed"
+        zIndex: 1000
     showGhost = (pageY) ->
+      $ghostContainer.appendTo("body").show()
+
       unless $ghost.is(":visible")
-        $ghost.addClass "sordidDragon-ghost"
-        $ghost.css
-          position: "fixed"
-          opacity: 1
-        $parent.append $ghost
+        $ghost
+          .addClass "sordidDragon-ghost"
+          .css
+            position: "fixed"
+            opacity: 1
+        $ghostContainer.append $ghost
+
       $ghost.css
         left: $placeholder.offset().left
         top: (pageY - ($placeholder.outerHeight() / 2)) - window.scrollY
         width: $placeholder.outerWidth() - window.scrollX
 
     hideGhost = ->
       $ghost?.remove()
+      $ghostContainer?.html("")
```

Comment:
May it be better to create the `$ghost` once then just show/hide it rather than destroying it (2 DOM ops) each time